api/catalog/api/docs/base_docs.py

```diff
@@ -0,0 +1,85 @@
+from http.client import responses as http_responses
+from textwrap import dedent
+
+from drf_spectacular.openapi import AutoSchema, OpenApiResponse
+from drf_spectacular.utils import OpenApiExample, extend_schema
+
+
+refer_curl = (
+    "You can refer to the cURL request samples "
+    "for examples on how to consume this endpoint."
+)
+
+
+def fields_to_md(field_names):
```

Comment:
Do you think it brings a lot of value to have a list of field names, if we give all of them below anyways?

Reply:
It's not very useful, I'd be willing to remove it if more people feel like it's not useful.